fix linking issue with libX11:
error adding symbols: DSO missing from command line
author: Kenneth Hoste (HPC-UGent)
--- luscus_0.8.6/CMakeLists.txt.orig	2018-11-23 15:01:47.737809689 +0100
+++ luscus_0.8.6/CMakeLists.txt	2018-11-23 15:01:54.027747459 +0100
@@ -124,7 +124,7 @@
   target_link_libraries(luscus ${GTKGLEXT_LDFLAGS} -lm)
 else(USE_GTK3)
   add_definitions(-DGTK2 -D_GNU_SOURCE ${GTK2_CFLAGS})
-  target_link_libraries(luscus ${GTK2_LDFLAGS} -lm)
+  target_link_libraries(luscus ${GTK2_LDFLAGS} -lm -lX11)
 endif(USE_GTK3)
 
 include_directories("${PROJECT_BINARY_DIR}")
